app/controllers/items_controller.rb

```diff
@@ -38,7 +36,13 @@ def update
     end
   end
 
+  def destroy
```

Comment:
Let's make a conditional branch in this items_controller as well, so that only the user who exhibited the item can delete it!
(If the conditional branch is only in view(show.html.erb), the delete action can be moved by using a validation tool, so it is impossible to deal with malicious users.)

【英語に自信がないので、日本語でも記述しておきます！m(_ _)m】
出品したユーザーだけが削除できるように、コントローラーでも条件分岐をしましょう！
（viewのみの条件分岐ですと、検証ツールを用いて削除のアクションを動かせてしまうため、
悪意のあるユーザーに対応できないため。）

Reply:
Thank you! Good English!

I added the conditional branch to destroy action so that checks if the user is the seller.